app/javascript/lib/honeycrisp.js

```diff
@@ -0,0 +1,323 @@
+var incrementer = (function() {
```

Comment:
This is a total copy paste port of the JS from honeycrisp. I've made a PR to make this an importable file from honeycrisp, but I don't know how quickly that will move: https://github.com/codeforamerica/honeycrisp-gem/pull/241

Reply:
ho ho ho, it's merged :) merry christmas!